controller/helpers.js

```diff
@@ -277,6 +277,19 @@ const checkEthBlockNumbers = async () => {
     console.log(JSON.stringify(res))
 }
 
+const handleBackups = async (callback, useBackupParams, backupParams) => {
+    try {
+        if (useBackupParams && backupParams) return await callback(backupParams);
+        return await callback();
+    } catch (error) {
+        if (backupParams) {
+          return await handleBackups(callback, true, backupParams);
+        } else {
+          throw error;
```

Comment:
I may be reading this recursive call wrong, but I do not see how it gets to "throw error". Has the case case where both servers fail been tested?

I can see this working if you use on line 285:

if (!useBackupParams) {

But that is strange. Maybe better to pass in "true" for useBackupParams when you first call it, then in line 286 change it to false so the backup recursion only gets called once.